pallets/collator-selection/README.md

```diff
@@ -1 +1,4 @@
-License: Unlicense
\ No newline at end of file
+License: Unlicense
```

Comment:
```suggestion
License: Apache 2.0
```

Reply:
`collator-selection` will be re-licensed to `Apache 2.0`? In `Cargo.toml` it's `Unlicense`.

Reply:
Not a relicense, Unilicense there is an error.

- Pallet: https://github.com/paritytech/statemint/blob/master/pallets/collator-selection/src/lib.rs#L2
- Repo: https://github.com/paritytech/statemint/blob/master/LICENSE
- FRAME: https://github.com/paritytech/substrate/blob/master/LICENSE-APACHE2